ui/analyse/src/study/relay/relayCtrl.ts

```diff
@@ -31,43 +27,33 @@ export default class RelayCtrl {
   showStreamerMenu = toggle(false);
   videoPlayer?: VideoPlayer;
   liveboardPlugin?: LiveboardPlugin;
-  readonly baseRedraw: Redraw;
-  readonly send: AnalyseSocketSend;
 
   constructor(
-    readonly id: RoundId,
+    private study: StudyCtrl,
```

Comment:
I tried quite hard to avoid the circular dependency, but yeah, it's just simpler with it...